### ISSUE TITLE
Change in the report

### ISSUE THREAD

**Issue by speedprovedor** [tananaev/traccar#3457]
Has to implicit these reports
Driver identifier
And direct sensor of the car sensor
  And decrease ram memory consumption

**Comment by tananaev** [comment]
What is this?

**Comment by speedprovedor** [comment]
The driver who is driving at the moment.

**Comment by tananaev** [comment]
Driver functionality is already implemented in the latest code.

**Comment by speedprovedor** [comment]
Which is the cheapest device that makes this identification

**Comment by tananaev** [comment]
I don't have answer for that question. You can find list of protocols that support this functionality like this:

https://github.com/tananaev/traccar/search?q=KEY_DRIVER_UNIQUE_ID&type=Code

**Comment by speedprovedor** [comment]
The app also takes a few seconds to initialize on the smartphone.

**Comment by tananaev** [comment]
That has nothing to do with this topic and this repo.

**Comment by speedprovedor** [comment]
I would not be able to do this indetermination via app

**Comment by speedprovedor** [comment]
And a suggestion an application to undelete by means of some tracker registration code

**Comment by speedprovedor** [comment]
And a suggestion an application to identify the driver by means of some tracker registration code

**Comment by tananaev** [comment]
I don't understand what you are talking about. Please write in proper English.

**Comment by speedprovedor** [comment]
Could identify the driver by application app traccar

**Comment by tananaev** [comment]
Are you asking if it's possible to identify driver using Traccar Client app? No, it's not possible at the moment.

**Comment by speedprovedor** [comment]
yes. yes  It's more and a good suggestion. Because the tracker with this function is very expensive

**Comment by tananaev** [comment]
As I said, it's a wrong thread and repo for the suggestion.
